src/web-auth/index.js

```diff
@@ -179,12 +179,14 @@ WebAuth.prototype.parseHash = function(options, cb) {
  */
 WebAuth.prototype.validateAuthenticationResponse = function(options, parsedHash, cb) {
   var _this = this;
+  options.__enableIdPInitiatedLogin =
```

Comment:
Minor but ... I wonder if you shouldn't call this something else since it encompasses 2 different use cases here? Like `options.__bypassStateChecking` or thereabouts?